lib/rules/selector-max-pseudo-class/__tests__/index.js

```diff
@@ -0,0 +1,186 @@
+"use strict";
+
+const messages = require("..").messages;
+const ruleName = require("..").ruleName;
+const rules = require("../../../rules");
+
+const rule = rules[ruleName];
+
+testRule(rule, {
+  ruleName,
+  config: [0],
+  skipBasicChecks: true,
+
+  accept: [
+    {
+      code: "",
+      description: "empty stylesheet"
+    },
+    {
+      code: "a {}",
+      description: "empty rule"
+    },
+    {
+      code: '@import "foo.css";',
+      description: "blockless statement"
+    },
+    {
+      code: "a {}",
+      description: "selector with no pseudo class"
```

Comment:
Please hyphenate [pseudo-classes](https://www.w3.org/TR/selectors-3/#pseudo-classes) in all the descriptions.